### DISCUSSION TITLE
My github education account is approved but the email has not been sent

### DISCUSSION THREAD

**Discussion by yash149** [community/community#133904] [GitHub Education]
### Hi everyone,

My GitHub education account is approved but the email has not been sent and as for the rejection letter it came just after the submission of the form and I have very less time as I have to develop a project and submit it to the school and the marks given will be added in my portfolio

**Comment by samus-aran** [comment]
Hi @yash149 👋 

Can you confirm how many Rejection Emails you have received and what dates?

I can confirm that you are currently in the Approved state so you should receive your academic benefits usually within 72 hours of that approval. However this may take a little longer depending on the volume of other applications.

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
